go/install/fuse_status_default.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2015 Keybase, Inc. All rights reserved. Use of
 // this source code is governed by the included BSD license.
 
-// +build !darwin,!windows
+// +build !darwin,!windows ios
 
 package install
 
```

Comment:
Is this file related to the build break, or did it sneak into the commit?

Reply:
Not related but I think it's the correct thing to do since there's no fuse on mobile

Reply:
Ah ok, so `!darwin` was excluding iOS before, but we want to include it? (Meanwhile `android` was already implicitly included?)

Reply:
Correct. Kinda weird because ios was already excluded in: https://github.com/keybase/client/blob/master/go/install/fuse_status_darwin.go#L4